index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="wrapper">
+    <nav class="menu">
+      <div class="logo">
+        <h3>Welcome</h3>
+      </div>
+      <div class="hamburger">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+
+    </nav>
+    <section id="headline">
+      <header>
+        <h2 class="darkColor">Hey there. I'm Gabriel</h2>
+        <h2 class="greenColor">I'm a software developer</h2>
+      </header>
+      <p> I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see
+        and have a project you need coded, don’t hesitate to contact me.</p>
+        <div class="socialMedia">
+          <img src="assets/images/twitter.svg" alt="twitter icon">
+          <img src="assets/images/linkedin.svg" alt="linkedin icon">
+          <img src="assets/images/medium.svg" alt="medium icon">
+          <img src="assets/images/github.svg" alt="github icon">
+          <img src="assets/images/angellist.svg" alt="angelist icon">
+        </div>
```

Comment:
- [ ] Please make sure that this list of social icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) for semantics reasons.
[Hint]: For example:
```html
<ul>
    <li>
       <a href="add link here"><img src="assets/images/twitter.svg" alt="twitter icon"></a>
    </li>
    ..............
    ..............
</ul>
```

### Other than that you have done a great job 💪.

Reply:
Hi Omar, should I add only the social media I use?? and add real links to them?

Reply:
Hi, 
kindly add the links available for you and leave the other links empty.

Reply:
Ok, thanks a lot for the feedback and quick response.